bedrock/settings/static_media.py

```diff
@@ -630,41 +630,22 @@
     },
     'home': {
         'source_filenames': (
-            'css/mozorg/home.less',
-            'css/mozorg/home-promo.less',
+            'css/mozorg/home/home.less',
+            'css/mozorg/home/home-promo.less',
         ),
         'output_filename': 'css/home-bundle.css',
     },
-    'home-ie9': {
-        'source_filenames': (
-            'css/mozorg/home-ie9.less',
-        ),
-        'output_filename': 'css/home-ie9-bundle.css',
-    },
-    'home-ie8': {
-        'source_filenames': (
-            'css/mozorg/home-ie8.less',
-        ),
-        'output_filename': 'css/home-ie8-bundle.css',
-    },
     'home-ie': {
```

Comment:
Is this bundle still needed? I don't see it included in any templates. If not, let's drop it and remove `home-ie.less`.